config/sync/views.view.media_library.yml

```diff
@@ -1261,11 +1374,11 @@ display:
           label: Table
           plugin_id: display_link
           empty: true
-      css_class: 'media-library-view js-media-library-view media-library-view--widget'
+      css_class: ''
+      rendering_language: '***LANGUAGE_language_interface***'
```

Comment:
Huh? Is this right?

Reply:
ya i had to check that too. https://git.drupalcode.org/project/drupal/blob/8.8.x/core/modules/media_library/config/install/views.view.media_library.yml#L1090